src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -1,14 +1,77 @@
+/*
+ * File: z_en_ru2.c
+ * Overlay: En_Ru2
+ * Description:
+ */
+
 #include "z_en_ru2.h"
 
 #define ROOM 0x00
 #define FLAGS 0x00000010
 
-void EnRu2_Init(EnRu2* this, GlobalContext* globalCtx);
-void EnRu2_Destroy(EnRu2* this, GlobalContext* globalCtx);
-void EnRu2_Update(EnRu2* this, GlobalContext* globalCtx);
-void EnRu2_Draw(EnRu2* this, GlobalContext* globalCtx);
+static void EnRu2_Init(EnRu2* this, GlobalContext* globalCtx);
+static void EnRu2_Destroy(EnRu2* this, GlobalContext* globalCtx);
+static void EnRu2_Update(EnRu2* this, GlobalContext* globalCtx);
+static void EnRu2_Draw(EnRu2* this, GlobalContext* globalCtx);
+
+static void func_80AF2CB4(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF2CD4(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF2CF4(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF2D2C(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF2D6C(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF2DAC(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF2DEC(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3144(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3174(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF31C8(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3604(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3624(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF366C(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF36AC(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3BC8(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3C04(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3C64(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3CB8(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3D0C(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3D60(EnRu2* this, GlobalContext* globalCtx);
+
+static void func_80AF3F14(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3F20(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF321C(EnRu2* this, GlobalContext* globalCtx);
+
+static void func_80AF2AB4(EnRu2* this, GlobalContext* globalCtx);
+
+static ColliderCylinderInit D_80AF40E0 = {
```

Comment:
I suggest entirely avoiding braces inside cylinder init data until we have a proper way to define it, and just let clang-format do its thing.